superchain/configs/mainnet/op.toml

```diff
@@ -29,25 +29,25 @@ seq_window_size = 3600
     gasLimit = 30000000
 
 [addresses]
```

Comment:
All of the `addresses` diffs shown in this PR are due to reordering the fields. Previously they were alphabetically ordered, but the toml package we are using orders them by their placement in the struct. The `just codegen` command run as part of CI confirms that none of the addresses written to `superchain/extra/addresses/addresses.json` have changed